op-challenger/config/config_test.go

```diff
@@ -132,122 +172,134 @@ func TestCannonRequiredArgs(t *testing.T) {
-				cfg := validConfig(traceType)
+				cfg := validConfig(t, traceType)
 				cfg.Cannon.SnapshotFreq = 0
 				require.ErrorIs(t, cfg.Check(), ErrMissingCannonSnapshotFreq)
 			})
 		})
 
 		t.Run(fmt.Sprintf("TestCannonInfoFreq-%v", traceType), func(t *testing.T) {
 			t.Run("MustNotBeZero", func(t *testing.T) {
-				cfg := validConfig(traceType)
+				cfg := validConfig(t, traceType)
 				cfg.Cannon.InfoFreq = 0
 				require.ErrorIs(t, cfg.Check(), ErrMissingCannonInfoFreq)
 			})
 		})
 
 		t.Run(fmt.Sprintf("TestCannonNetworkOrRollupConfigRequired-%v", traceType), func(t *testing.T) {
-			cfg := validConfig(traceType)
+			cfg := validConfig(t, traceType)
 			cfg.Cannon.Network = ""
 			cfg.Cannon.RollupConfigPath = ""
 			cfg.Cannon.L2GenesisPath = "genesis.json"
 			require.ErrorIs(t, cfg.Check(), ErrMissingCannonRollupConfig)
 		})
 
 		t.Run(fmt.Sprintf("TestCannonNetworkOrL2GenesisRequired-%v", traceType), func(t *testing.T) {
-			cfg := validConfig(traceType)
+			cfg := validConfig(t, traceType)
 			cfg.Cannon.Network = ""
 			cfg.Cannon.RollupConfigPath = "foo.json"
 			cfg.Cannon.L2GenesisPath = ""
 			require.ErrorIs(t, cfg.Check(), ErrMissingCannonL2Genesis)
 		})
 
 		t.Run(fmt.Sprintf("TestMustNotSpecifyNetworkAndRollup-%v", traceType), func(t *testing.T) {
-			cfg := validConfig(traceType)
+			cfg := validConfig(t, traceType)
 			cfg.Cannon.Network = validCannonNetwork
 			cfg.Cannon.RollupConfigPath = "foo.json"
 			cfg.Cannon.L2GenesisPath = ""
 			require.ErrorIs(t, cfg.Check(), ErrCannonNetworkAndRollupConfig)
 		})
 
 		t.Run(fmt.Sprintf("TestMustNotSpecifyNetworkAndL2Genesis-%v", traceType), func(t *testing.T) {
-			cfg := validConfig(traceType)
+			cfg := validConfig(t, traceType)
 			cfg.Cannon.Network = validCannonNetwork
 			cfg.Cannon.RollupConfigPath = ""
 			cfg.Cannon.L2GenesisPath = "foo.json"
 			require.ErrorIs(t, cfg.Check(), ErrCannonNetworkAndL2Genesis)
 		})
 
 		t.Run(fmt.Sprintf("TestNetworkMustBeValid-%v", traceType), func(t *testing.T) {
-			cfg := validConfig(traceType)
+			cfg := validConfig(t, traceType)
 			cfg.Cannon.Network = "unknown"
 			require.ErrorIs(t, cfg.Check(), ErrCannonNetworkUnknown)
 		})
 
 		t.Run(fmt.Sprintf("TestNetworkMayBeAnyChainID-%v", traceType), func(t *testing.T) {
-			cfg := validConfig(traceType)
+			cfg := validConfig(t, traceType)
 			cfg.Cannon.Network = "467294"
 			require.NoError(t, cfg.Check())
 		})
 
 		t.Run(fmt.Sprintf("TestNetworkInvalidWhenNotEntirelyNumeric-%v", traceType), func(t *testing.T) {
-			cfg := validConfig(traceType)
+			cfg := validConfig(t, traceType)
 			cfg.Cannon.Network = "467294a"
 			require.ErrorIs(t, cfg.Check(), ErrCannonNetworkUnknown)
 		})
 
 		t.Run(fmt.Sprintf("TestDebugInfoEnabled-%v", traceType), func(t *testing.T) {
-			cfg := validConfig(traceType)
+			cfg := validConfig(t, traceType)
 			require.True(t, cfg.Cannon.DebugInfo)
 		})
+
+		t.Run(fmt.Sprintf("TestCannonVMBinExists-%v", traceType), func(t *testing.T) {
```

Comment:
(nit) Minor nit here - you can drop the vm-specific naming for these new tests since we are formatting them with the `traceType`:
```suggestion
		t.Run(fmt.Sprintf("TestVMBinExists-%v", traceType), func(t *testing.T) {
```